### PR TITLE
Example of two groups with two instances of ind_rob_sim.

### PR DESCRIPTION
**Pull request by gavanderhoorn**
Just an example.

With `motoman_driver`, things will be similar, but not exactly the same.

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
Just saw your adaptation of this PR in a334efc0 and it doesn't seem correct.

Especially the "real" `controllers.yaml` file that is missing the two additional controller definitions.

File: config/controllers.yaml [hunk 1]
```diff
@@ -1,20 +1,20 @@
 controller_list:
-  - name: ""
+  # This controller will exist when motoman_driver is used, but it's not easily
+  # simulated with industrial_robot_simulator.
+  #
+  # Note: I 'made up' the 'dx200' name(space).
+  #
+  # - name: dx200
```
Comment (tdl-ua) [inline config/controllers.yaml:7]: @gavanderhoorn Do I need to just uncomment this when I want to interact with a real controller and also leave the below two controllers (r1 and b1) in there?

Comment (gavanderhoorn) [reply config/controllers.yaml:7]: If you have two groups, then you'd have 3 entries here: one for each group + one for all joints.

**Comment by tdl-ua** [comment]
@gavanderhoorn besides the missing controllers in the `controllers.yaml` file, did you see any other issues?

**Pull request closed without merging**